### DISCUSSION TITLE
Is there a way to authenticate our Company users through the Customer login page/application?

### DISCUSSION THREAD

**Discussion by trapperl** [casdoor/casdoor#2376] [Q&A] [answered]
I intend to use Casdoor for a distributed Building Management System.

We have our Company users in an Organization/Application and we have each customer as their own Organization/Application.

Is there a way to authenticate our Company users through the Customer login page/application without adding our Company users to the Customers users?

**Comment by hsluoyz** [comment, marked as answer]
Not possible, organization is designed as different user pools, that's how it is designed
